store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,80 @@ TODO: delete them permanently from here
+</attr>
+
+<attr id="4010" name="zimbraBackupStartTime" type="string" max="64" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>01:00</globalConfigValue>
+  <desc>Starts backup each day at this time. This attribute is used to set hours and minutes on the crontab.</desc>
+</attr>
+
+<attr id="4011" name="zimbraBackupRetentionDays" type="integer" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>8</globalConfigValue>
+  <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value.</desc>
+</attr>
+
+<attr id="4012" name="zimbraBackupBlobsCompressType" type="enum" value="zip,zipStore,noZip" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>zip</globalConfigValue>
+  <desc>Blobs inside a backup are compressed by default in zip format.
+        zip - blobs are backed up into zip files with compression (default).
+        zipStore - blobs are backed up into zip files without compression.
+        noZip - blobs are backed up as individual files rather in zip files.
+  </desc>
+</attr>
+
+<attr id="4013" name="zimbraBackupSyncEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>If true, the full backup runs synchronously.
+        If false, full backup runs in a separate thread and the soap call returns with the backup label.
+  </desc>
+</attr>
+
+<attr id="4014" name="zimbraMailReportEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, sends an email report to the admin user. Admins can disable the email notification by setting it to FALSE.</desc>
+</attr>
+
+<attr id="4015" name="zimbraBackupStartDay" type="string" max="32" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>6</globalConfigValue>
+  <desc>The day of the week on which the full backup starts.
+        When a day is select their relative value is set on the crontab. Default starts from Saturday
+        0-7 or names allowed (Sun, Wed), with both 0 and 7 representing Sunday. Monday to Saturday (1-6)
+  </desc>
+</attr>
+
+<attr id="4016" name="zimbraBackupObjectLevelEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, --accounts option will be set to zimbraBackupConfigEnabled to zmschedulebackup command. Enables cos,domain,server,account level backup.
+        If false, --accounts option will be set to all accounts to zmschedulebackup command. Backups all accounts.
+  </desc>
+</attr>
```

Comment:
I think logic should be reverse, as this option says in UI "**backup all users**".
So, 
**If true**, --accounts option will be set to all accounts to zmschedulebackup command. Backups all accounts.
**If false**, --accounts option will be set to zimbraBackupConfigEnabled to zmschedulebackup command. Enables cos,domain,server,account level backup.

Reply:
I think it should be ok considering we are actually adding this object level backup as a new functionality and backing up "all users" is just a fall backup value. As mentioned in the other comments, we are considering these attributes to be generic and not specific to the UI.

Reply:
Isn't `useBackupConfig` (not `zimbraBackupConfigEnabled`) added if it is true?